docs/en/reference/transactions-and-concurrency.rst

```diff
@@ -114,8 +114,8 @@ functionally equivalent to the previously shown code looks as follows:
 The difference between ``Connection#transactional($func)`` and
 ``EntityManager#transactional($func)`` is that the latter
 abstraction flushes the ``EntityManager`` prior to transaction
-commit and rolls back the transaction when an
-exception occurs.
+commit and in case of an exception the ``EntityManager`` gets closed
+in addition to the transaction rollback.
```

Comment:
Maybe this formulation is more clear. What do you think about it?
```suggestion
after the transaction rollback.
```

Reply:
Sorry for late reply.

IMO there's no benefit to imply an order (EM gets closed **after** transaction rollback; is that even the correct order?) And for the user, it makes no difference for understanding the behavior. That's why I decided to write "in addition to", in contrast to the first part of the changed text where order matters.

But in the end it's a matter of taste 😃